src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -4,23 +4,85 @@
+    <Compile Include="System\Reflection\Context\Projection\IProjectable.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingAssembly.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingConstructorInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingCustomAttributeData.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingEventInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingExceptionHandlingClause.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingFieldInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingLocalVariableInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingManifestResourceInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingMethodBody.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingMethodInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingModule.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingParameterInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingPropertyInfo.cs" />
+    <Compile Include="System\Reflection\Context\Projection\ProjectingType.cs" />
+    <Compile Include="System\Reflection\Context\Projection\Projector.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\InheritedMethodInfo.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\InheritedPropertyInfo.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualMethodBase.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualParameter.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualProperty.PropertyGetter.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualProperty.PropertySetter.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualPropertyBase.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualPropertyBase.FuncPropertyAccessorBase.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualPropertyBase.PropertyGetterBase.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualPropertyBase.PropertySetterBase.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualPropertyInfo.cs" />
+    <Compile Include="System\Reflection\Context\Virtual\VirtualReturnParameter.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">
+    <Reference Include="System.Collections" />
+    <Reference Include="System.Diagnostics.Contracts" />
     <Reference Include="System.Reflection" />
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Security.Permissions" />
```

Comment:
You will need to add some references like System.Runtime for netstandard1.1 as well. That is why CI is failing.